cookbook/form/create_form_type_extension.rst

```diff
@@ -129,6 +129,9 @@ The ``alias`` key of the tag is the type of field that this extension should
 be applied to. In your case, as you want to extend the ``file`` field type,
 you will use ``file`` as an alias.
 
+If you want to create a generic feature, applied to all field types, simply
```

Comment:
I liked to have the original vocabulary or at least the word "generic" to make is searchable. But I like your formulation too and its location in the page should be enough.

New commit on its way.